src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -108,7 +108,7 @@ public function send(string $body, array $headers, int $delay = 0): string
         $now = new \DateTime();
         $availableAt = (clone $now)->modify(sprintf('+%d seconds', $delay / 1000));
 
-        $queryBuilder = $this->driverConnection->createQueryBuilder()
+        $queryBuilder = $this->createQueryBuilder()
```

Comment:
Why changing this? It's doing an insert, so the helper method seems useless here.